python/lsst/ts/electrometer/csc.py

```diff
@@ -1,4 +1,4 @@
-# This file is part of ts_electrometer.
+#This file is part of ts_electrometer.
```

Comment:
not sure what happened here... was this you or `black` ? if it's the latter then just ignore this.

Reply:
Oh shoot! I did run black and changed that but didn't push it. Doh.